corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -58,19 +58,15 @@ def __init__(self, domain, app_id, version, key_lang, source_lang, lang_prefix,
         self._build_translations()
 
     @cached_property
-    def _get_labels_to_comments(self):
-        labels_to_comments = {}
+    def _get_labels_to_skip(self):
+        labels_to_skip = []
         module_data, errors = get_form_data(self.domain, self.app)
         for module in module_data:
-            form_dict = defaultdict(list)
-
             for form in module['forms']:
                 for question in form['questions']:
-                    if question['comment'] and 'label_ref' in question:
-                        form_dict[question['label_ref']].append(question['comment'])
-                labels_to_comments[form['id']] = form_dict
-
-        return labels_to_comments
+                    if 'SKIP TRANSIFEX' in question['comment'] and 'label_ref' in question:
+                        labels_to_skip.append(question['label_ref'])
+        return labels_to_skip
```

Comment:
Shouldn't `labels_to_skip` be per form?  That is, couldn't multiple forms have questions with the same label?

Reply:
Sorry, yes. Thank you. Fixed.